userdetails-cognito/src/main/groovy/au/org/ala/userdetails/CognitoApplicationService.groovy

```diff
@@ -150,6 +151,7 @@ class CognitoApplicationService implements IApplicationService {
         if (applicationRecord.type == ApplicationType.GALAH) {
             request.callbackURLs.addAll(galahCallbackURLs)
         }
+        request.callbackURLs.addAll(tokensCallbackURLs)
```

Comment:
I'm not sure we should be adding tokens callback URLs to all apps?  Maybe add a checkbox to allow the user to opt-in?

Reply:
Fixed